modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -119,25 +119,22 @@ You can define scopes in three different places:
 ...
 ----
 
-Full https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform] asset available on Exchange.
+For more information, see https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform].
 
 == Client Application Cache
 
-To prevent downtime of the Mule OAuth 2.0 provider due to errors when connecting to the platform,
-the Mule OAuth Client Store caches each valid Client Application for which a token is requested.
+To prevent any possible downtime of the Mule OAuth 2.0 provider, the Mule OAuth Client Store caches each valid Client Application for which a token is requested. The downtime can occur due to errors when connecting to Anypoint Platform.
 
-If the Mule OAuth 2.0 provider requests a new token while there is no connection, the OAuth 2.0 Client Store falls back to the cache.
+If the Mule OAuth 2.0 provider requests a new token during no connection, the OAuth 2.0 Client Store falls back to the cache. The entries expire in 30 days, which is the default setting. To ensure service availability in case of a restart or lack of connection to Anypoint Platform, you can configure this range to persist the cache locally.
```

Comment:
During no connection, seems to imply that it is an expected occurrence, which is not the case.

Reply:
Kim please verify.

Reply:
"If" implies that this can happen and, if it does, what the result is.
It does not imply expectation but nor does it imply rarity.

If you want to imply that lack of connection is _rare_, consider something like "In the rare instance that the provider requests a new token _when there is no connection available_, the client store falls back to cache."

"during no connection" sounds awkward; reword.

Reply:
"In the rare instance that.." is ok. Perhaps using "unlikely" instead of "rare"?

Reply:
Done! Thanks!